### ISSUE TITLE
Please clarify the meaning of "applied" in relation to ID Token algorithm selection

### ISSUE THREAD

**Issue by pcurtisrab** [cdr-register/register#97]
In verstion 1.1.1 of the standard there is a section titled "ID Token Algorithm Selection Considerations".

Are the four algorithms mentioned here somehow binding on participants? If so, how?
- Must Data Holders support all algorithms?
- Must Data Recipients support all algorithms?
- May Data Holders support other algorithms?
- May Data Recipients support other algorithms?

**Comment by CDR-Register-Stream** [comment]
@pcurtisrab Thank you for your query.

The section you are referring to was introduced in version [1.1.0](https://cdr-register.github.io/register/includes/releasenotes/releasenotes.1.1.0.html#v1-1-0-release-notes). 

The [DCR request](https://cdr-register.github.io/register/#registration-request-using-jwt) documentation was updated to include the following ID Token algorithms currently being applied in the ecosystem:

Claim | Values
-- | --
id_token_encrypted_response_alg | RSA-OAEP</br>RSA-OAEP-256
id_token_encrypted_response_enc | A256GCMA</br>128CBC-HS256

The CDS does not mandate what algorithms must be supported and defers to the upstream standards. However, it would be preferable to have only a subset of algorithms used in the ecosystem to reduce the ADR implementation effort. In order to provide guidelines to participant in the ecosystem, we have highlighted what algorithms are currently being used by participants.

The onus is on the ADRs to support as many algorithms as the ecosystem is using in order to be able to communicate with all Data Holders.

So...
* Data Holders should support at least one of the algorithms in each list
* Data Recipients should support all the algorithms used in the ecosystem to ensure they can communicate with all Data Holders.
